app/src/main/java/org/schabi/newpipe/player/helper/PlayerHolder.java

```diff
@@ -69,14 +83,19 @@ public static void setListener(final PlayerServiceExtendedEventListener newListe
         }
     }
 
-    public static void removeListener() {
+    public void removeListener() {
```

Comment:
Remove this and let `setListener` receive a `@Nullable` parameter. When the parameter is `null` the listener should just be removed.

Reply:
you want me to remove the method removeListener() completly and replace it with a call to setListener(null);
Is that what you meant?

Reply:
Yes